app/services/ScanningService.scala

```diff
@@ -20,10 +20,14 @@ import model.UploadedFile
 
 import scala.concurrent.Future
 
+sealed trait ScanningResult
```

Comment:
As discussed, it would be good to make a ScanningResult contain a S3ObjectLocation, as logically the two are coupled concepts and are used in conjunction.